src/Viewer/components/Monaco/MonacoInstance.js

```diff
@@ -272,22 +257,23 @@ function MonacoInstance ({
         };
     }, []);
 
-    const monacoOptions = {
-        "readOnly": true,
-        "renderWhitespace": "none",
-        "wordWrap": "on",
-        "scrollBeyondLastLine": false,
-    };
+    useEffect(() => {
+        if (null === editorRef.current) {
```

Comment:
Change L222: 
```
if (editorRef && editorRef.current)
```
->
```
if (null != editorRef.current)
```

editorRef is always an object and therefore always not null.

Reply:
!= or !== ?

Reply:
My aplologies - should be `!==`. Thanks for correcting another occurance!